ticket_dispenser.c

```diff
@@ -1,149 +1,203 @@
-#include <Wire.h>
-
+#include "Wire.h"
 #include "Keypad.h"
 #include "Adafruit_LEDBackpack.h"
 #include "Adafruit_GFX.h"
 
 Adafruit_7segment matrix = Adafruit_7segment();
-
+//
 // Setup the keypad
-const byte ROWS = 4; //four rows
-const byte COLS = 3; //three columns
+//
+const byte ROWS = 4;
+const byte COLS = 3;
 char keys[ROWS][COLS] = {
```

Comment:
Since this is a matrix, I wonder if you could build this programmatically rather than just using it declaratively. If so, you could potentially make the code smart enough to know about the inputs available to it, rather than telling it explicitly what it has available to it.